### PR TITLE
Feature/5 mettre une estimation de la précision de localisation

### PR DESCRIPTION
**Pull request by SarahHCode**
# Pull Request

## Summary
Made map user location's circle radius variable with "capacitor's geolocation accuracy"!

## Changes Made
Turned the user location icon to 'style' on location layer and added accuracy radius in realistic meters.

## Related Issues
~~!!! Thought about making the radius really in meters on the map because "capacitor geolocation accuracy" returns accuracy in meters, but not sure it's a good idea because if the radius is always too big (or too small), it could be ugly on the map and bothering to user... so thought about indicating accuracy in terms of proportion represented by the outer circle radius of the map user location icon, but not sure how... (the more the accuracy is big, the bigger the radius of outer circle)~~ Now put in realistic meters (added polygon circular which has radius in meters in OpenLayers), but without a limit on radius.
Also, how do I test if capacitor geolocation accuracy works (updates and returns accuracy)?

## Checklist
Please ensure all the following steps are completed before submitting the pull request:

- [ ] Code passes all existing tests
- [ ] New features or changes are accompanied by appropriate tests
- [ ] Code follows the established coding style and conventions
- [ ] Documentation has been updated to reflect the changes (if applicable)
- [ ] All new and existing tests pass locally

### CODE REVIEW

**Comment by kim841** [comment]
Je suis stuck dans la page de chargement, car il ne retrouve pas la fonction getAccuracy
image

**Comment by SarahHCode** [comment]
Oh, j'ai oublié d'ajouter les modifications dans UserData.ts dans le commit.
J'avais juste ajouté public static getAccuracy(){ return this.data.location.accuracy}.

File: src/components/MapContainer.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div id="map" class="map">
+    <!-- <p>{{test}}</p> -->
```
Comment (kim841) [inline src/components/MapContainer.vue:3]: peut être supprimé

**Review by kim841 — CHANGES_REQUESTED** [review]

**Comment by kim841** [comment]
2 petits bugs
- à certain moment lorsque je zoom, le cercle bleu transparent disparait
- sur un physical device, le cercle bleu transparent n'apparait pas

**Review by kim841 — APPROVED** [review]

**Comment by kim841** [comment]
Tu as fait du bon travail! Le code fonctionne bien!

Pour ce qui est de l'interface, il faudra voir avec Barbara si l'apparence du cercle est acceptable à ce niveau de zoom. Pour moi ça semble bon!
image

**Pull request merged**